src/components/menu/MenuList.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { color } from "../../shared/style";
+import MenuItem from "./MenuItem";
+import Sponsor from "../sponsor/Sponsor";
+
+const VALUES = {
+  TOOLS: "the tools",
+  LEARN: "learn more",
+  ABOUT: "about & contact",
+};
+
+const StyledList = styled.ul`
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  display: flex;
+
+  @media (max-width: 48rem) {
+    flex-flow: column nowrap;
+    border-top: 2px solid ${color.primary};
+    position: fixed;
+    top: 4.5rem;
+    bottom: 0;
+
+    ${({ open }) =>
+      open
+        ? `& {
+          opacity: 1;
+          transition: opacity 200ms 0ms;
+        }`
+        : `& { 
+          opacity: 0;
+          transition: opacity 200ms 0ms;
+        }`};
+
+    & > li {
+      width: 100%;
+      ::after {
+        border: none;
+      }
+    }
+  }
+`;
+
+const MenuList = ({ open }) => {
+  const [selected, setSelected] = useState(VALUES.TOOLS);
+  return (
+    <StyledList aria-label="menu list" open={open}>
+      {Object.values(VALUES).map(item => (
+        <MenuItem
+          key={item}
+          value={item}
+          onSelect={() => setSelected(item)}
+          isSelected={item === selected}
+          text={item}
+        />
+      ))}
+      <Sponsor />
```

Comment:
```suggestion
      <li>
        <Sponsor />
      </li>
```

This needs to be wrapped with `li`-tags, as `<ul>` and `<ol>` must only directly contain `<li>`, `<script>` or `<template>` elements.

Reply:
Done.